examples/IdentityServerIntegration/IdentityServer.fs

```diff
@@ -0,0 +1,99 @@
+﻿namespace Suave.IdentityServer
+
+module Security =  
```

Comment:
If there's only a single module, why not just have it under `Suave.IdentityServer` directly?